app/models/miq_report/seeding.rb

```diff
@@ -68,14 +68,15 @@ def sync_from_file(filename, dir, typ)
       rpt[:template_type] = typ
       rec = self.find_by_filename(rpt[:filename])
 
+      type_titleized = typ.titleize
```

Comment:
In the existing code conversion happens already just once (if/else). You're adding an extra (unused) conversion in case `rec.file_mtime.utc >= rpt[:file_mtime]`